dspace/config/modules/storage.cfg

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------#
+#-----------------STORAGE CONFIGURATIONS------------------------#
+#---------------------------------------------------------------#
+# Configuration properties used by the bitstore.xml config file #
```

Comment:
It's confusing to me that this file includes everything *except* `assetstore.dir` (which is the main configuration needed for `localStore`).

Should we consider renaming this file to be `assetstore.cfg` (as all configs start with `assetstore.*` and also MOVING `assetstore.dir` to this file?  We could simply leave a comment in `dspace.cfg` where `assetstore.dir` exists that says: 

```
# Assetstore configurations have moved to config/modules/assetstore.cfg
# and config/spring/api/bitstore.xml.
```